modules/jdbc-test/src/test/java/org/testcontainers/junit/SimplePostgreSQLTest.java

```diff
@@ -16,24 +16,48 @@
  * @author richardnorth
  */
 public class SimplePostgreSQLTest {
-
-    @Rule
-    public PostgreSQLContainer postgres = new PostgreSQLContainer();
-
     @Test
     public void testSimple() throws SQLException {
+        PostgreSQLContainer postgres = new PostgreSQLContainer();
+        postgres.start();
+
+        try {
+            ResultSet resultSet = performQuery(postgres,"SELECT 1");
+            int resultSetInt = resultSet.getInt(1);
+            assertEquals("A basic SELECT query succeeds", 1, resultSetInt);
+        } finally {
+            postgres.stop();
+        }
+    }
+
+    @Test
+    public void testCommandOverride() throws SQLException {
+        PostgreSQLContainer postgres = (PostgreSQLContainer) new PostgreSQLContainer().withCommand("postgres -c max_connections=42");
```

Comment:
No need for the cast here, right?

Reply:
Required (at least to have `PostgreSQLContainer` as the type) 🤷‍♀️ :
```
SimplePostgreSQLTest.java:36: error: incompatible types: GenericContainer cannot be converted to PostgreSQLContainer
        PostgreSQLContainer postgres = new PostgreSQLContainer().withCommand("postgres -c max_connections=42");
```

Reply:
Yikes.  Sorry for the trouble.

Reply:
This comes about because of generic typing (the withCommand method is on GenericContainer). The need to cast should go away if we use `new PostgreSQLContainer<>()`. It looks like some other tests use the cast for some unknown reason, but we really should have to have it anywhere.